src/hooks/useKeyboardArrows.js

```diff
@@ -39,3 +39,19 @@ export function useEsc(cb) {
     return () => window.removeEventListener('keydown', handlekeyDown)
   }, [handlekeyDown])
 }
+
+export function useEnterKey(cb, validated) {
```

Comment:
Even if it means another `useCallback()` in the component itself, I think we should leave the `validated` bit to the callback handler—best not to add too much surface area to this hook.

Reply:
not sure if i follow the idea, i am pretty sure that a hook can not be called inside a useCallback

Reply:
Ah, what I meant here is that we should keep the API of `useEnteryKey()` as

```
function useEnterKey(cb)
```

And let the component handle its own validation logic—even if it means inside that component, there's a small event handler wrapped with `useCallback()`.